### PR TITLE
OCPBUGS#32056: Clarify the `global-operators` Operator group exists

### PR DESCRIPTION
**Pull request by michaelryanpeter**
Version(s): 4.12+

Issue: [OCPBUGS-32056](https://issues.redhat.com/browse/OCPBUGS-32056)

Link to docs preview:

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Review by bandrade — APPROVED** [review]
/lgtm

File: modules/olm-installing-from-operatorhub-using-cli.adoc [hunk 1]
```diff
@@ -73,13 +73,14 @@ $ oc describe packagemanifests <operator_name> -n openshift-marketplace
 
 . An Operator group, defined by an `OperatorGroup` object, selects target namespaces in which to generate required RBAC access for all Operators in the same namespace as the Operator group.
 +
-The namespace to which you subscribe the Operator must have an Operator group that matches the install mode of the Operator, either the `AllNamespaces` or `SingleNamespace` mode. If the Operator you intend to install uses the `AllNamespaces`, then the `openshift-operators` namespace already has an appropriate Operator group in place.
+The namespace to which you subscribe the Operator must have an Operator group that matches the install mode of the Operator, either the `AllNamespaces` or `SingleNamespace` mode. If the Operator you intend to install uses the `AllNamespaces` mode, then the `openshift-operators` namespace already has the appropriate `global-operators` Operator group in place.
```
Comment (maxwelldb) [inline modules/olm-installing-from-operatorhub-using-cli.adoc:76]: Don't really need the "then" here. ISG discourages it.

**Review by maxwelldb — APPROVED** [review]
Looks good. Made one comment that isn't strictly within the scope of your changes.

File: modules/olm-installing-from-operatorhub-using-cli.adoc (hunk 1 above)
Comment (michaelryanpeter) [reply modules/olm-installing-from-operatorhub-using-cli.adoc:76]: Good call. It also reads better without it.

**Pull request merged**